app/editor/src/components/form/select/Select.tsx

```diff
@@ -81,7 +86,13 @@ export const Select = <OptionType extends IOptionItem>({
           {label}
         </label>
       )}
-      <Row>
+      <Row
+        onKeyUp={(e) => {
+          if (e.code === 'Delete') {
+            onClear?.();
```

Comment:
This does not work great, but it does work enough that it's possible to clear the selected value.  It would be nice to get it working better.